tests/Fixer/Import/NoUnusedImportsFixerTest.php

```diff
@@ -705,16 +705,6 @@ class Ddd
 //Foo
 #Bar
 /*Baz*/',
-            ],
-            'with_case_insensitive_matches_in_comments' => [
```

Comment:
You have just remove 1 test, please elaborate why this one is incorrect, as it looks fine.

Also, instead of removing test case, please, replace it with one that will test your changes and prevent regression in the future.

Reply:
I realized my mistake before you replied to me, and I supplemented the test again. Thank you.